elements/pfe-tabs/src/pfe-tab-panel.scss

```
@@ -37,7 +46,7 @@ $LOCAL-VARIABLES: (
     @include browser-query(ie11) {
       padding: 1em;
       background-color: #{map-get($pfe-colors, surface--lightest)} !important;
-      color: #{pfe-var(text)} !important;
+      color: #{pfe-fetch(text)} !important;
```

```suggestion
      color: pfe-fetch(text) !important;
```

Interpolation is not needed here

---

updated